devito/finite_differences/differentiable.py

```diff
@@ -157,30 +148,39 @@ def __getattr__(self, name):
         raise AttributeError("%r object has no attribute %r" % (self.__class__, name))
 
     # Override SymPy arithmetic operators
+    @call_highest_priority('__radd__')
```

Comment:
I guess this is a bug fix, and it's getting tested..

Reply:
It's tested through the tensor tests yes. Was done manually before but basically this enforce that it always uses the add/mul for the one with the highest priority between self and other.